content/glossary/_index.md

```diff
@@ -2,311 +2,411 @@
 
-### Ticket
+## Ticket or VRF Chain
 
-A `ticket` is used as a source of randomness in EC leader election. Every block depends on one or more `ChallengeTicket` derived from a `PartialTicket` using input from a `RandomnessTicket` (naming PR welcome). One such new `RandomnessTicket` or `Ticket` is produced with every new block and included in its header
+Tickets are generated as in [Ticket](glossary#ticket) above, but the input includes the concatenation of the previous ticket. This means that the new ticket is generated running the VRF on the old ticket concatenated with the new DRAND value (and the key as with Ticket).
 
-### Ticket Chain
+## Tipset
 
-Each chain in Filecoin can be associated with a given `ticket chain`. The `ticket chain` is assembled by taking the tickets (usually one) contained by the block with the smallest final ticket in each of the chain's `TipSet`s.
+A [tipset](https://filecoin.io/blog/tipsets-family-based-approach-to-consensus/) is a set of [blocks](glossary#block) that each have the same [height](glossary#block-height) and parent tipset; the Filecoin [blockchain](glossary#blockchain) is a chain of tipsets, rather than a chain of blocks.
 
-Ticket comparison is done by interpreting the tickets' Bytes as unsigned integers (little endian representation).
+Each tipset is assigned a weight corresponding to the amount of storage the network is provided per the commitments encoded in the tipset's blocks. The consensus protocol of the network directs nodes to build on top of the heaviest chain.
 
-### TipSet
+By basing its blockchain on tipsets, Filecoin can allow multiple [storage miners](glossary#storage-miner-actor) to create blocks in the same [epoch](glossary#epoch), increasing network throughput. By construction, this also provides network security: a node that attempts to intentionally prevent the valid blocks of a second node from making it onto the canonical chain runs up against the consensus preference for heavier chains.
 
-A `TipSet` is a set of blocks that have the same parent set and same number of `tickets`, which implies they will have been mined at the same `height`. A `TipSet` can contain multiple blocks if more than one miner successfully mines a block at the same `height` as another miner.
+## Verified client
 
+To further incentivize the storage of "useful" data over simple [capacity commitments](glossary#capacity-commitment), [storage miners](glossary#storage-miner-actor) have the additional opportunity to compete for special [deals](glossary#deal) offered by verified clients. Such clients are certified with respect to their intent to offer deals involving the storage of meaningful data, and the power a storage miner earns for these deals is augmented by a multiplier.
```

Comment:
Do we have a section explaining verified client? Asking cause the Verified Client concept came about after the initial writing of the spec